### ISSUE TITLE
translation timer

### ISSUE THREAD

**Issue by Gyglov** [Danily07/Translumo#69]
Add a translation timer, so that where you can't set permanent text appearance, the log doesn't get clogged. I'm getting tired of pushing start-stop translation.

**Comment by Danily07** [comment]
Hi! The feature for auto-clear translation window after specified delay has been implemented in latest release [0.9.5](https://github.com/Danily07/Translumo/releases/tag/v.0.9.5). Or I got you wrong and did you mean something else?

**Comment by Gyglov** [comment]
No, I'm not talking about auto-cleaning of the window, I'm talking about the frequency of text scanning, you can't customize it.

**Comment by Danily07** [comment]
Oh, I see. But I still didn't get how it could help you. If the app has an additional delay, it will also affects _real_ text which you want to translate (delayed translation is quite inconvenient)

**Comment by Gyglov** [comment]
Why is it so difficult? Not everywhere you can set the speed at which text appears, and so that the window is not clogged with the translation of incomplete text, you can set a timer to scan the text. That is, the text appears in 3 seconds and set the timer for 3 seconds, and not like now where it scans with an interval of 0.5 seconds.

**Comment by Danily07** [comment]
I didn't say that it's difficult to implement :) Just I can't clearly grasp profit based on understanding how algorithm really works. But okay, I will add the task to backlog and implement that via additional configuration parameter in config file (don't want to overload UI with secondary settings)

**Comment by Agress0r** [comment]
Я поддерживаю это улучшение, дабы тот же EasyOCR считывал текст в указанной области но уже с интервалом который указал ты сам, это поможет производительности при его использовании, но при этом качество останется.
